bundles/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/handler/AirQualityStationHandler.java

```diff
@@ -110,13 +110,13 @@ public void initialize() {
     private void discoverAttributes() {
         getAirQualityData().ifPresent(data -> {
             // Update thing properties
-            Map<String, String> properties = new HashMap<>();
-            properties.put(ATTRIBUTIONS, data.getAttributions());
+            Map<String, String> properties = new HashMap<>(Map.of(ATTRIBUTIONS, data.getAttributions()));
```

Comment:
Isn't it ok to do only
```
Map<String, String> properties = Map.of(ATTRIBUTIONS, data.getAttributions());
```
?

Reply:
No because properties would then be immutable and I need the ability to add some other entries.